Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1342,6 +1492,42 @@ def identityFS(wf):
 steps['MINIAODMCUP15FS']   =merge([{'--filein':'file:step1.root','--fast':''},stepMiniAODMC])
 steps['MINIAODMCUP15FS50'] =merge([{'--conditions':'auto:run2_mc_50ns','--era':'Run2_50ns'},steps['MINIAODMCUP15FS']])
 
+## re-miniAOD for Prod tests
+steps['REMINIAOD'] = merge([{'--conditions':'auto:run2_mc_'+autoHLT['relval25ns'],
+                                   '--runUnscheduled':'',
+                                   '-s':'PAT,DQM:@miniAODDQM,VALIDATION:@miniAODValidation',
+                                   '--datatier' : 'MINIAODSIM,DQMIO',
+                                   '--eventcontent':'MINIAODSIM,DQM',},stepMiniAODMC])
+
+
+steps['REMINIAOD_PU25'] = steps['REMINIAOD']
+steps['REMINIAOD_PU50'] = merge([{'--conditions':'auto:run2_mc_'+autoHLT['relval50ns'],'--era':'Run2_50ns'},steps['REMINIAOD']])
+
+
+## re-miniAOD for Prod tests
+steps['REMINIAODPROD']=merge([{
+                    '-s':'PAT,DQM:@miniAODDQM,VALIDATION:@miniAODValidation',
+                    #'-s':'PAT,DQM:@miniAODDQM',
+                    '--datatier':'MINIAODSIM,DQMIO',
+                    '--eventcontent':'MINIAODSIM,DQM'
+                    },step3Up2015Defaults])
+
+
+### re-miniAOD for data 2015
+
+REMINIAODDR2={ '--runUnscheduled':'',
+          '--conditions':'auto:run2_data_'+menuR2_25ns,
+          '-s':'PAT,DQM:@miniAODDQM',
+          '--datatier':'MINIAOD,DQMIO',
+          '--eventcontent':'MINIAOD,DQM',
+          '--data':'',
+          '--scenario':'pp',
+          }
+
+steps['REMINIAODDR2_25ns']=merge([{'--conditions':'auto:run2_data_'+menuR2_25ns,'--customise':'Configuration/DataProcessing/RecoTLR.customiseDataRun2Common_25ns',},REMINIAODDR2])
```

Comment:
@hengne 
I proposed to use era instead of the customization.
If it's still clear that era is not applicable for cmsDriver-build miniAOD-only workflow, then OK, we can live with the customizations longer.